code/game/objects/structures/fitness.dm

```diff
@@ -0,0 +1,118 @@
+/obj/structure/punching_bag
+	name = "punching bag"
+	desc = "A punching bag. Can you get to speed level 4???"
+	icon = 'icons/obj/fitness.dmi'
+	icon_state = "punchingbag"
+	anchored = 1
+	var/list/hit_sounds = list('sound/weapons/genhit1.ogg', 'sound/weapons/genhit2.ogg', 'sound/weapons/genhit3.ogg',\
+	'sound/weapons/punch1.ogg', 'sound/weapons/punch2.ogg', 'sound/weapons/punch3.ogg', 'sound/weapons/punch4.ogg')
+
+/obj/structure/punching_bag/attack_hand(mob/user as mob)
+	flick("[icon_state]2", src)
+	playsound(loc, pick(hit_sounds), 25, 1, -1)
+
+/obj/structure/punching_bag/wizard
+	icon_state = "punchingbagwizard"
+	desc = "It has a picture of a weird wizard on it."
+
+/obj/structure/punching_bag/syndie
+	icon_state = "punchingbagsyndie"
+	desc = "It has a picture of a mean ol' syndicate on it."
+
+/obj/structure/punching_bag/captain
+	icon_state = "punchingbagcaptain"
+	desc = "It has a picture of a dumb looking station captain on it."
+
+/obj/structure/punching_bag/clown
+	name = "clown bop bag"
+	desc = "A bop bag in the shape of a goofy clown."
+	icon_state = "bopbag"
+
+/obj/structure/punching_bag/clown/attack_hand(mob/user as mob)
+	flick("[icon_state]2", src)
+	playsound(loc, pick(hit_sounds), 25, 1, -1)
+	playsound(loc, 'sound/items/bikehorn.ogg', 50, 1, -1)
+
+/obj/structure/stacklifter
+	name = "weight machine"
+	desc = "Just looking at this thing makes you feel tired."
+	icon = 'icons/obj/fitness.dmi'
+	icon_state = "fitnesslifter"
+	density = 1
+	anchored = 1
+
+/obj/structure/stacklifter/attack_hand(mob/user as mob)
+	if(in_use)
+		to_chat(user, "<span class='notice'>Its already in use - wait a bit.</span>")
+		return
+	else
+		in_use = 1
+		icon_state = "fitnesslifter2"
+		user.change_dir(SOUTH)
+		user.Stun(4)
+		user.forceMove(loc)
+		var/bragmessage = pick("pushing it to the limit","going into overdrive","burning with determination","rising up to the challenge", "getting strong now","getting ripped")
+		user.visible_message("<B>[user] is [bragmessage]!</B>")
+		var/lifts = 0
+		while (lifts++ < 6)
```

Comment:
I'm so sorry but now I'm getting "missing left-hand argument to <." when I changed the two lines to
		for(var/lifts = 1; lifts++; lifts =< 6)